### ISSUE TITLE
Only 3 Mount-Points ?

### ISSUE THREAD

**Issue by Just-In-Web** [AzuraCast/AzuraCast#791]
Installation Method: Docker on Ubuntu 18.04

I can only set 3 Mount-Points for Icecast Servers. I can add more but they won't start. 

What can I do?

**Comment by SlvrEagle23** [comment]
@Just-In-Web What do you mean by "won't start"? They don't appear at all on the profile page? Or they show up, but they don't pick up any audio from the AutoDJ?

**Comment by Just-In-Web** [comment]
@SlvrEagle23 I cann add it and it's showing in the profile but there is nothing. On the requested Mount-Point it shows "The file you requested could not be found".

**Comment by Just-In-Web** [comment]
![image](https://user-images.githubusercontent.com/23245228/45545985-35711a80-b81c-11e8-9a36-f8f6217f0391.png)

Mount-Page


![image](https://user-images.githubusercontent.com/23245228/45546019-50438f00-b81c-11e8-9b24-c688db8a8b76.png)

Icecast Status.

Restart is already done.

**Comment by SlvrEagle23** [comment]
@Just-In-Web Thank you for the updates. I looked into the configuration, and it appears that the `<sources>3</sources>` parameter in the Icecast configuration is for how many _total_ sources can be connected, not necessarily how many _unique_ sources. In this case, even Liquidsoap connecting 4 times counts as 4 sources, which threw that number off.

I've updated the configuration to set that sources limit automatically based on the number of mountpoints, which should resolve the issue entirely.
